packages/css/src/components/card/card.template.ts

```diff
@@ -0,0 +1,43 @@
+import { Card } from '@dso-toolkit/sources';
+import { html, TemplateResult } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+import { buttonTemplate } from '../button/button.template';
+import { iconTemplate } from '../icon/icon.template';
+
+export function cardTemplate({ label, content, interactions }: Card<TemplateResult>) {
+  return html`
+    <div class="dso-card">
+      <div class="dso-card-heading">
+        <a href="#">
+          <div class="dso-rich-content">
```

Comment:
Is `dso-rich-content` hier nog wel te rechtvaardigen? Nu er 'n onderdeel `dso-card-heading` is bedacht, moet de `dso-rich-content` dan niet eerder op/in `dso-card-content` komen?

Een heading lijkt mij uit een duidelijke structuur te bestaan, `dso-rich-content` is er voor de situaties dat die structuur op voorhand niet bekend is